src/core/src/op/paged_attention.cpp

```diff
@@ -220,6 +285,8 @@ void PagedAttentionExtension::validate_and_infer_types() {
 }
 
 std::shared_ptr<ov::Node> PagedAttentionExtension::clone_with_new_inputs(const ov::OutputVector& new_args) const {
+    OV_OP_SCOPE(_PagedAttentionExtension_clone_with_new_inputs);
```

Comment:
```suggestion
    OV_OP_SCOPE(PagedAttentionExtension_clone_with_new_inputs);
```
Probably leftover from deleting v16 opset